packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -1,165 +1,306 @@
+    },
+    text: {
+      primary: 'var(--joy-palette-neutral-800)',
+      secondary: 'var(--joy-palette-neutral-600)',
+      tertiary: 'var(--joy-palette-neutral-500)',
+    },
+    background: {
+      default: 'var(--joy-palette-neutral-50)',
+      level1: '#fff',
+      level2: 'var(--joy-palette-neutral-100)',
+      level3: 'var(--joy-palette-neutral-200)',
+    },
+    focusVisible: 'var(--joy-palette-primary-200)',
+  },
+  elevationRing: '0 0 #000',
+};
+
+export const darkColorSystem: Pick<BaseJoyTokens, 'palette' | 'elevationRing'> = {
   palette: {
-    brand: colors.purple,
-    neutral: colors.grey,
+    primary: {
+      ...colors.purple,
+      ...createDarkModeVariantVariables('primary'),
+    },
+    neutral: {
+      ...colors.grey,
+      ...createDarkModeVariantVariables('neutral'),
+    },
+    danger: {
+      ...colors.red,
+      ...createDarkModeVariantVariables('danger'),
+    },
+    info: {
+      ...colors.blue,
+      ...createDarkModeVariantVariables('info'),
+    },
+    success: {
+      ...colors.green,
+      ...createDarkModeVariantVariables('success'),
+    },
+    warning: {
+      ...colors.yellow,
+      ...createDarkModeVariantVariables('warning'),
+    },
     text: {
-      heading: 'var(--joy-palette-neutral-900)',
-      headingIntro: 'var(--joy-palette-brand-300)',
-      content: 'var(--joy-palette-neutral-600)',
-      detail: 'var(--joy-palette-neutral-500)',
-      overline: 'var(--joy-palette-neutral-500)',
+      primary: 'var(--joy-palette-neutral-100)',
+      secondary: 'var(--joy-palette-neutral-300)',
+      tertiary: 'var(--joy-palette-neutral-400)',
     },
-    bgNeutral: {
-      transparency: 'var(--joy-palette-neutral-100)',
-      plain: 'var(--joy-palette-neutral-100)',
+    background: {
+      default: 'var(--joy-palette-neutral-800)',
+      level1: 'var(--joy-palette-neutral-900)',
+      level2: 'var(--joy-palette-neutral-700)',
+      level3: 'var(--joy-palette-neutral-600)',
     },
+    focusVisible: 'var(--joy-palette-primary-400)',
   },
+  elevationRing: '0 0 #000',
+};
+
+/**
+ * Base Joy design tokens
+ * Any value with `var(--joy-*)` can be used. 'joy-' will be replaced by the application prefix if provided.
+ */
+const joyDesignTokens: BaseJoyTokens = {
+  ...lightColorSystem,
   borderRadius: {
-    md: '4px',
+    default: '28px',
+    xs: '4px',
+    sm: '8px',
+    md: '12px',
+    lg: '20px',
+    xl: '32px',
+  },
+  elevation: {
+    xs: 'var(--joy-elevationRing), 0 1px 2px 0 rgba(0, 0, 0, 0.05)',
+    sm: 'var(--joy-elevationRing), 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06)',
+    md: 'var(--joy-elevationRing), 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06)',
+    lg: 'var(--joy-elevationRing), 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)',
+    xl: 'var(--joy-elevationRing), 0 20px 25px -5px rgba(0, 0, 0, 0.1), 0 10px 10px -5px rgba(0, 0, 0, 0.04)',
```

Comment:
```suggestion
  elevation: {
    xs: 'var(--joy-elevationRing), 0 1px 2px 0 rgba(0, 0, 0, 0.05)',
    sm: 'var(--joy-elevationRing), 0.3px 0.8px 1.1px rgba(187, 187, 187, 0.11), 0.5px 1.3px 1.8px -0.6px rgba(187, 187, 187, 0.18), 1.1px 2.7px 3.8px -1.2px rgba(187, 187, 187, 0.26)',
    md: 'var(--joy-elevationRing), 0.3px 0.8px 1.1px rgba(187, 187, 187, 0.12), 1.1px 2.8px 3.9px -0.4px rgba(187, 187, 187, 0.17), 2.4px 6.1px 8.6px -0.8px rgba(187, 187, 187, 0.23), 5.3px 13.3px 18.8px -1.2px rgba(187, 187, 187, 0.29)',
    lg: 'var(--joy-elevationRing), 0.3px 0.8px 1.1px rgba(187, 187, 187, 0.11), 1.8px 4.5px 6.4px -0.2px rgba(187, 187, 187, 0.13), 3.2px 7.9px 11.2px -0.4px rgba(187, 187, 187, 0.16), 4.8px 12px 17px -0.5px rgba(187, 187, 187, 0.19), 7px 17.5px 24.7px -0.7px rgba(187, 187, 187, 0.21)',
    xl: 'var(--joy-elevationRing), 0.3px 0.8px 1.1px rgba(187, 187, 187, 0.11), 1.8px 4.5px 6.4px -0.2px rgba(187, 187, 187, 0.13), 3.2px 7.9px 11.2px -0.4px rgba(187, 187, 187, 0.16), 4.8px 12px 17px -0.5px rgba(187, 187, 187, 0.19), 7px 17.5px 24.7px -0.7px rgba(187, 187, 187, 0.21), 10.2px 25.5px 36px -0.9px rgba(187, 187, 187, 0.24), 14.8px 36.8px 52.1px -1.1px rgba(187, 187, 187, 0.27), 21px 52.3px 74px -1.2px rgba(187, 187, 187, 0.29)',
```

Done with the help of [Josh Cameau's shadow palette generator](https://www.joshwcomeau.com/css/introducing-shadow-palette-generator/) :)